### ISSUE TITLE
Different img_norm_cfg std values for the PASCAL VOC dataset in SSD and faster_rcnn scripts

### ISSUE THREAD

**Issue by hyz-xmaster** [open-mmlab/mmdetection#2416]
Hi there, thanks for building this great toolbox. 

I was trying to train my detector on the PASCAL VOC dataset. 

I noticed that the image normalization parameter - **std** values in [ssd512_voc.py](https://github.com/open-mmlab/mmdetection/blob/365c9302ee1eb5790b8e57f24ea4dfee8f2b88ac/configs/pascal_voc/ssd512_voc.py#L49) and [ssd300_voc.py](https://github.com/open-mmlab/mmdetection/blob/365c9302ee1eb5790b8e57f24ea4dfee8f2b88ac/configs/pascal_voc/ssd300_voc.py#L49)
`img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[1, 1, 1], to_rgb=True)`
are different from those in [faster_rcnn_r50_fpn_1x_voc0712.py ](https://github.com/open-mmlab/mmdetection/blob/365c9302ee1eb5790b8e57f24ea4dfee8f2b88ac/configs/pascal_voc/faster_rcnn_r50_fpn_1x_voc0712.py#L104-L105)
`img_norm_cfg = dict(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)`.

So if I use **pretrained='open-mmlab://resnet50_caffe', the std should be [1.0, 1.0, 1.0]**, while when **pretrained='torchvision://resnet50', the std becomes [58.395, 57.12, 57.375]**? Is this assumption correct?

Thank you.

**Comment by xvjiarui** [comment]
Hi @hyz-xmaster 
You are right.

**Comment by hyz-xmaster** [comment]
Thanks.
